### ISSUE TITLE
Better defaults for createdDate

### ISSUE THREAD

**Issue by akshaymankar** [scubajeff/lespas#162]
**Is your feature request related to a problem? Please describe.**

I'm always frustrated when I move/copy pictures I received in Whatsapp to albums because they always get the createdDate as the day I am moving them instead of the day I received them. In the gallery I see them at the date they were sent to me, as soon as I move/copy them the date changes. 

**Describe the solution you'd like**

I am guessing this happens because Whatsapp removes any EXIF data when sending the image, perhaps using the same way to figure out the date would be good which is used to show the images in the gallery.

**Describe alternatives you've considered**

I've considered remembering to move images to albums as soon as I receive them, but its too hard to follow this. 

**Additional context**

None.

**Comment by scubajeff** [comment]
I don't use Whatsapp, but I guess it behaves the same way like all the other social app, e.g
, removes EXIF before sending pictures. Could you give me an example of what's the picture filename you received? Or even better, send me a file you received to my email scubajeffrey@protonmail.ch?

**Comment by akshaymankar** [comment]
The images are named like `IMG-20240203-WA0001.jpg` and videos are named like `VID-20240203-WA0001.mp4`.

**Comment by scubajeff** [comment]
Obviously only date coded in. It will be hard to guess the time. Is the filename pattern customizable in whatsapp?

**Comment by akshaymankar** [comment]
I don't think there is a way to customize the filename pattern. For me using the date without the time (or defaulting that to midnight) would already be an improvement.

**Comment by scubajeff** [comment]
Ok. Fix will be included in next release.
